### DISCUSSION TITLE
About Adding Apu skin temp limit

### DISCUSSION THREAD

**Discussion by RAGE-HERO7** [seerge/g-helper#2688] [General]
Hello @seerge i noticed you added apu skin temp limit in the prerelease version, i think adding that option is a bad idea, i just searched on internet and chatgpt to know about the consequence of tuning that option and got this as answer. I am truly worried people gonna unknowingly play around with this option and break there systems.

My suggestion is to give people this option only available after modifying config.jason manually for those who need

![Screenshot_20240605_013111_ChatGPT](https://github.com/seerge/g-helper/assets/154586512/26e2b7b3-a24d-46b5-a3a0-9269bc84d1a3)

If my findings are wrong and if i am being unnecessarily worried about this please do correct me :)

**Comment by seerge** [comment]
@RAGE-HERO7 hello, that option existed before as well. I have just added a separate slider for it (check original topic https://github.com/seerge/g-helper/issues/2673 )

This is needed for literally opposite purpose.

**Reply by RAGE-HERO7** [reply]
@seerge Pardon my ignorance,  i should have read #2673 before starting the discussion, i understand the need for Apu temp limit option, Thank you for adding it 👍😊

**Reply by seerge** [reply]
Actually, since you messaged. I have checked it a bit more and it doesn't seem to work for me. 

So probably makes sense to hide it, but not because it can do something, but cause it just does nothing :)

**Reply by RAGE-HERO7** [reply]
lol! , in that case if you are proceeding with making this option available for all
i think you gonna hear lots of people complaining about the option not working for there devices. like back in the old version you made the undervolting option available for all AMD cpu.
just predicting 😅

**Reply by seerge** [reply]
Yes, general approach is to show only those features - that are applicable for each specific model, and hide unsupported. This applies to undervolting as well. Even tho people still complain on reddit that "G-Helper does not show undervolting for me ... "
